Allow build.prod.rollup.aot for multiple apps

<!--
IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MIGHT CLOSE YOUR ISSUE WITHOUT INVESTIGATING
-->

**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see use [gitter](https://gitter.im/mgechev/angular2-seed) or [stackoverflow](https://stackoverflow.com/questions/tagged/angular2)
```

**Current behavior**
<!-- Describe how the bug manifests. -->
When I run `build.prod.rollup.aot` with `-- --app other`, I get 

`dist/tmp/app/main-prod.ts(8,36): error TS2307: Cannot find module './app.module.ngfactory'.` 

in `build.js.prod.rollup.aot`.

**Expected behavior**
<!-- Describe what the behavior would be without the bug. -->
Rollup AOT build works.

Comparing with `build.js.prod.aot`, the `toIgnore` section is missing in `build.js.prod.rollup.aot`. When this section is copied, the build works for me.

**Angular Seed Version:** `aaaaf75`
<!-- Check which is the hash of the last commit from angular-seed that you have locally -->

--- a/tools/tasks/seed/build.js.prod.rollup.aot.ts
+++ b/tools/tasks/seed/build.js.prod.rollup.aot.ts
@@ -1,3 +1,4 @@
+import { readdirSync, lstatSync } from 'fs';
 import * as gulp from 'gulp';
 import * as gulpLoadPlugins from 'gulp-load-plugins';
 import { join } from 'path';

--- a/tools/tasks/seed/build.js.prod.rollup.aot.ts
+++ b/tools/tasks/seed/build.js.prod.rollup.aot.ts
@@ -17,9 +17,15 @@ export = () => {
     module: 'es2015',
     moduleResolution: 'node'
   }, Config.TMP_DIR);
+
+  let toIgnore = readdirSync(Config.TMP_DIR).filter((f: string) =>
+    lstatSync(join(Config.TMP_DIR, f)).isDirectory() && f !== Config.BOOTSTRAP_DIR)
+    .map((f: string) => '!' + join(Config.TMP_DIR, f, Config.NG_FACTORY_FILE + '.ts'));
+
   const src = [
     join(Config.TOOLS_DIR, '/manual_typings/**/*.d.ts'),
-    join(Config.TMP_DIR, '**/*.ts')
+    join(Config.TMP_DIR, '**/*.ts'),
+    ...toIgnore
   ];
   const result = gulp.src(src)
     .pipe(plugins.plumber())

